packages/flutter_tools/doc/daemon.md

```diff
@@ -230,3 +242,4 @@ See the [source](https://github.com/flutter/flutter/blob/master/packages/flutter
 ## Changelog
 
 - 0.3.0: Added `daemon.connected` event at startup
+- 0.4.0: Added `emulator.create` command
```

Comment:
[nit] In most other change logs I've seen, the most recent entry comes first.

[Mostly curious] Do you have a strategy for how to do `0.x.y` versioning? Some Dart package authors seem to bump `x` only for breaking changes, `y` for anything else, effectively treating x as a major version number until the package graduates to `1.0.0`.

Reply:
I did write it the other way, but it felt a bit weird since it was only part of the doc and a single line for each. I was kinda on the fence though, so I'll swap it around; I think you're right that it probably makes more sense the other (especially as it grows).

I don't think there's a solid strategy here, but I figured this was a "new feature" so it felt bigger than a bug fix. The use of this version is fairly simple though - probably nobody uses it today except the code I'm writing just now in Dart Code which does a >= 0.4.0 check to know whether the daemon will support creating emulators. I don't think it would be a good idea for an editor to ever do <= checks (we wouldn't want to just disable Flutter functionality saying "this daemon is too new and we don't understand it" - the user provides the Flutter version, we don't get to choose it) so probably a single increasing integer would've worked here.

Again; I don't mind changing if anyone has a preference; all that's important is that it's increased so I know when the daemon I'm talking to supports creating :-)

Reply:
Thanks. I don't have a strong opinion. Your call.